src/screens/Settings/Account.jsx

```diff
@@ -128,6 +171,58 @@ export default function Account() {
         <GoogleSignin text="Connect to Google" />
       )}
 
+      <Title s2 className={styles.span2}>
+        Time Zone
```

Comment:
I would make it "Change your timezone"

Reply:
Wouldn't that break with the current style? All other titles in the settings aren't phrased actions.

Reply:
I think that's okay! Throwing some sass in there every now and then 👍